compiler/rustc_expand/src/mbe/macro_rules.rs

```diff
@@ -263,9 +263,7 @@ fn generic_extension<'cx, 'tt>(
 
                 // Ignore the delimiters on the RHS.
                 let rhs = match &rhses[i] {
-                    mbe::TokenTree::Delimited(_, delimited) => {
-                        delimited.inner_tts().to_vec().clone()
-                    }
+                    mbe::TokenTree::Delimited(_, delimited) => delimited.tts.to_vec().clone(),
```

Comment:
```suggestion
                    mbe::TokenTree::Delimited(_, delimited) => delimited.tts.to_vec(),
```

Reply:
Also the results of `open_tt`/`close_tt` are sometimes cloned unnecessarily now.

Reply:
Can you point me at some? I thought I removed all the clones on `open_tt`/`close_tt`.

Reply:
Ah, ok, indeed.
I've seen them in my local codebase, but didn't notice that they are removed in this PR.

Reply:
👍